libdnf/module/ModulePackageContainer.hpp

```diff
@@ -209,6 +209,7 @@ struct ModulePackageContainer
     std::vector<ModulePackagePtr> query(std::string subject);
     std::vector<ModulePackagePtr> query(std::string name, std::string stream,
         std::string version, std::string context, std::string arch);
+    void enableDependencyTree(std::vector<ModulePackagePtr> modulePackages);
```

Comment:
Do we really need a copy of the vector?
void enableDependencyTree(const std::vector<ModulePackagePtr> & modulePackages);